Source/WebKit/WebProcess/WebCoreSupport/WebStorageConnection.cpp

```diff
@@ -44,17 +44,17 @@ Ref<WebStorageConnection> WebStorageConnection::create()
 
 void WebStorageConnection::getPersisted(WebCore::ClientOrigin&& origin, StorageConnection::PersistCallback&& completionHandler)
 {
-    connection().sendWithAsyncReply(Messages::NetworkStorageManager::Persisted(origin), WTFMove(completionHandler));
+    Ref { connection() }->sendWithAsyncReply(Messages::NetworkStorageManager::Persisted(origin), WTFMove(completionHandler));
```

Comment:
Should `connection()` return `Ref` instead?

(This does not need to be fixed to land this patch.)

Reply:
For now, we're not doing that in cases where you want to do something like connection().foo().bar() where foo() and bar() are trivial functions.